pkg/operator/etc/config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	HealthProbeBindAddress        string        `env:"OPERATOR_HEALTH_PROBE_BIND_ADDRESS" envDefault:":9090"`
 	CISKubernetesBenchmarkEnabled bool          `env:"OPERATOR_CIS_KUBERNETES_BENCHMARK_ENABLED" envDefault:"true"`
 	VulnerabilityScannerEnabled   bool          `env:"OPERATOR_VULNERABILITY_SCANNER_ENABLED" envDefault:"true"`
+	VulnerabilityScannerReportTTL string        `env:"OPERATOR_VULNERABILITY_SCANNER_REPORT_TTL" envDefault:""`
```

Comment:
We can use time.Duration and it will get parsed automatically. See ScanJobTimeout property.